lib/slimmer/app.rb

```diff
@@ -28,10 +28,7 @@ def initialize(app, *args, &block)
         options[:asset_host] = Plek.current.find("static")
       end
 
-      cache = Cache.instance
-      cache.use_cache = options[:use_cache] if options[:use_cache]
```

Comment:
Yeah, `use_cache` is definitely `false` in the smartanswers:

```
tijmenbrommet@production-calculators-frontend-1:~$ govuk_app_console smartanswers
irb(main):001:0> Slimmer::Cache.instance
=> #<Slimmer::Cache:0x007fb87ee53150 @cache_last_reset=2017-02-22 09:15:09 +0000, @use_cache=false, @cache_ttl=300, @data_store={}>
```

But I don't understand why since [it is looks like its turned on in production](https://github.com/alphagov/smart-answers/blob/master/config/initializers/slimmer.rb#L4-L6).

The same is true for `collections`:

```
irb(main):002:0> Slimmer::Cache.instance
=> #<Slimmer::Cache:0x007ff393677e68 @cache_last_reset=2017-02-22 09:21:01 +0000, @use_cache=false, @cache_ttl=300, @data_store={}>
```

But not `government-frontend`:

```
irb(main):004:0> Slimmer::Cache.instance
=> #<Slimmer::Cache:0x007f54a340cde8 @cache_last_reset=2017-02-22 09:21:42 +0000, @use_cache=true, @cache_ttl=300, @data_store={}>
```

😕